scripts/docker/README.md

```diff
@@ -0,0 +1,76 @@
+# Sandia-OpenSHMEM Docker Image
+Sandia-OpenSHMEM (SOS) utlizes Docker images to provide users a means to create
+containers that provide a configurable Ubuntu sandbox for running Sadia
+OpenSHMEM (SOS) applications and benchmarks.
+
+## Creating SOS Containers from the Prebuilt Docker Images
+Coming Soon: Information regarding the SOS public dockerhub repos and prebuilt
+images.
+
+## Creating SOS Containers from the Dockerfile 
+This Dockerfile provides a configurable Ubuntu sandbox for running Sadia
+OpenSHMEM (SOS) applications and benchmarks.
+
+### Building the Image from the Dockerfile
+To build the image from the Dockerfile, run the following run the following
+command in the directory that contains the Dockerfile:
+
```

Comment:
Minor edit comment: same words have been used with different capitalizations. For example, `Image` and `image`, `Dockerfile` and `dockerhub`.. I am not sure which one to use. just mentioning it if we need to correct / unify anything here.